app/upload/views.py

```diff
@@ -52,7 +53,16 @@ def upload_document(service_id):
         if not virus_free:
             return jsonify(error="File did not pass the virus scan"), 400
 
-    document = document_store.put(service_id, file_data, mimetype=mimetype)
+    if request.json.get("verification_email", False):
```

Comment:
With the default `False` value here, it's now possible for us to be dealing with three types: `None`, `str`, and `bool`. It would be nice to reduce this so that we have to think about fewer edge cases - can we just deal with `Optional[str]` instead (ie `None` and `str`)?

I guess this API doesn't have any json schema guarding types - I might look at adding one so that it's really clear what we expect.

Reply:
Yeah, I don't know why I put `False`, should be `None` 😳 . Too much writing stories, too little coding recently it seems :P . Thanks for spotting!